_pages/algorithm-development/source-control-management.md

```diff
@@ -136,6 +136,18 @@ Once you've created your algorithm, any commits to your GitHub repository's _def
 
 You can view your algorithm's builds by heading to it's landing page and clicking the "Builds" tab if you are the algorithm's owner.
 
+#### Publishing GitHub-Hosted Algorithms
+
+When you're ready to create a new version of your algorithm, visit that algorithm's page and select the __Builds__ tab. Find the Git hash of the build you'd like to use, and then click __Publish__ on the right side of the row.
```

Comment:
:bulb: Might tweak this phrasing to "Locate the build corresponding to specific commit SHA you wish to use, and then click **Publish** on the right side of the row."